service-front/module/Application/view/application/pages/donor_id_check.phtml

```diff
@@ -0,0 +1,100 @@
+<div class="govuk-phase-banner">
+    <p class="govuk-phase-banner__content">
+        <strong class="govuk-tag govuk-phase-banner__content__tag">
+            Beta
+        </strong>
+        <span class="govuk-phase-banner__text">
+      This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.
+    </span>
+    </p>
+</div>
+<div class="govuk-width-container">
+    <main class="govuk-main-wrapper">
+        <h3 class="govuk-heading-xl">Donor ID check</h3>
+        <h3 class="govuk-heading-m">Is the donor prepared to perform the ID check?</h3>
+
+        <p class="govuk-body">
+            Things to check:
+        </p>
+        <ul class="govuk-list govuk-list--bullet">
+            <li>
+                Which document would they prefer to ID with?
+            </li>
+            <li>
+                do they have the ID document with them?
+            </li>
+            <li>
+                Check the ID matches what is in Sirius (update as appropriate)
+            </li>
+        </ul>
+
+        <article class="moj-ticket-panel" aria-label="Sub navigation 1">
+            <section class="moj-ticket-panel__content moj-ticket-panel__content--blue" aria-label="Section 1">
```

Comment:
I'm not sure why the example in the design system uses article/section and aria-labels, but they don't semantically makes sense here so we should just use divs.
```suggestion
        <div class="moj-ticket-panel">
            <div class="moj-ticket-panel__content moj-ticket-panel__content--blue">
```